bioinf_modules/add_modules/fastq.py

```diff
@@ -0,0 +1,20 @@
+def count_gc(seq: str):
+    c = seq.count('C')
+    g = seq.count('G')
+    return (c + g)/len(seq)*100
```

Comment:
Можно сделать 1 проход по последовательности вместо 2х, это может немного ускорить алгоритм, если будет много данных
```suggestion
def count_gc(seq: str):
    gc_count = 0
    for base in seq:
        if base in "GC":
            gc_count += 1
    return gc_count / len(seq) * 100
```